packages/ns/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "mongodb-ns",
+  "version": "2.4.3",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "description": "MongoDB namespace parsing and validation",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "scripts": {
+    "test": "mocha",
+    "check": "mongodb-js-precommit"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mongodb-js/ns.git"
+  },
+  "keywords": [
+    "mongodb",
+    "mongodb.js"
+  ],
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint-config-mongodb-js": "^5.0.3",
+    "mocha": "^7.0.0",
+    "mongodb-js-precommit": "^2.0.0"
+  }
+}
```

Comment:
This works, but honestly, I'd recommend just using the create-workspace script to create this directory and then copy the README + index.js + index.d.ts + test.js into the created directory. That's what I've done in the past and it's not too much effort but is overall much nicer because then you end up with a package that's aligned with our general workflow, coding style, etc.

Optional but I'd definitely consider it

Reply:
ah that's what we meant by workspace! let me try that now

Reply:
Yeah, we're adopting the terminology of npm and similar tools here: https://docs.npmjs.com/cli/v8/using-npm/workspaces